yt_dlp/extractor/vidyard.py

```diff
@@ -0,0 +1,256 @@
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    extract_attributes,
+    int_or_none,
+    traverse_obj,
+)
+
+
+class VidyardBaseIE(InfoExtractor):
+
```

Comment:
If applying the suggestion below:
```suggestion

    _HEADERS = {}

```

Reply:
Referrer handling extracted as suggested as of b8345dd2076859b85857a82e010d4b793978b041